OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+2. Set the [Proxy](#10-how-to-fix-yumoraclecom-connectivity-error) if required.
+3. Assign Docker permission to your username. To assign, execute the command as **root** user:
+
+```
+    $ /sbin/usermod -a -G docker <nis unix userid>
+```
+   **Note**: To verify if the user has the Docker permission, see [FAQ](#12-permission-denied-while-connecting-to-the-docker-daemon-socket) section.
+
+## 4. Downloading Docker Images and Oracle WebCenter Sites Binary
+Before you begin creating an Oracle WebCenter Sites image on Docker, download the following images of Oracle Fusion Middleware Infrastructure, Oracle Database, and Oracle WebCenter Sites binaries. WebCenter Sites is installed on Oracle Fusion Middleware infrastructure, and it needs Oracle Database for storage.
+ 
+To download images from Oracle Container Registry (OCR) and Oracle Technology Network (OTN):
+
+Sign in to [OCR](https://container-registry.oracle.com). Click the **Sign in** link that's on the top-right of the Web page.
+
+### A. To download and set up Oracle Fusion Middleware Infrastructure image.
+1. Click **Home > Middleware** and then click **Continue** for the <i>fmw-infrastructure</i> repository.
+2. Click **Accept** to accept the license agreement.
+3. To download Oracle Fusion Middleware infrastructure base image:    
+    ```
+        $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3
+    ```
+	
+    Retag the Oracle Fusion Middleware Infrastructure Docker image by running the following command:
+    ```
+        $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
+    ```
+    **Note**: 
+    - If you want to download image from the Docker store, see [FAQ](#7-alternate-download-location-for-oracle-fusion-middleware-infrastructure-and-oracle-database-images) section.
+    - If you want to build image from the GitHub, see [FAQ](#9-how-do-i-build-an-oracle-fusion-middleware-infrastructure-1221x-base-image) section.
+
+### B. To download and set up Oracle Database Enterprise Edition image.
+1. Click **Home > Database** and then click **Continue** for <i>enterprise</i> repository.
+2. Click **Accept** to accept the license agreement.
+3. From the terminal, execute the following commands:
+       
+    Download Oracle Database 12.2.0.1 base image:    
+    ``` 
+        $ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1    
+    ```
+    Retag the Oracle Database Docker image by running the following command:
+    ```
+       $ docker tag container-registry.oracle.com/database/enterprise:12.2.0.1 database/enterprise:12.2.0.1
+    ```
+    **Note**: 
+    - If you want to download image from the Docker store, see [FAQ](#7-alternate-download-location-for-oracle-fusion-middleware-infrastructure-and-oracle-database-images) section.
+    - If you want to build image from the GitHub, see [FAQ](#8-how-do-i-build-an-oracle-database-1221x-base-image) section.
```

Comment:
You need to move a word in this sentence. It should be, "If you want to build the image from GitHub"

Reply:
Changes incorporated into latest file.